data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java

```diff
@@ -67,33 +67,31 @@ public GetValidators(final ChainDataProvider chainDataProvider, final JsonProvid
   @OpenApi(
       path = ROUTE,
       method = HttpMethod.GET,
-      summary = "Returns validators that match the specified query.",
+      summary = "Get validators that match the specified query.",
       tags = {TAG_BEACON},
       description =
-          "Returns validator information. If no parameters specified, the first page of validators as of the current epoch are returned.",
+          "Returns validator information. Returns the first page of validators in the current epoch if you do not specify any parameters.",
```

Comment:
can i buy some line breaks? if you use intellij just pressing 'enter' inside the string will close the string, add a '+' and restart the string on the next line...